### DISCUSSION TITLE
Close dialogue box on drop event?

### DISCUSSION THREAD

**Discussion by AndyOooh** [react-dropzone/react-dropzone#1225] [Q&A]
Hi, 
I'm using the hook.
The dialogue box remains open when I drop a file into the dropzone. Is this expected behavior?
If so, is there a way around it?

**Comment by rolandjitsu** [comment]
@AndyOooh an example would help so I can understand what you're trying to achieve or what you're experiencing.

But if I understand correctly, you click to open the native file section window, then you're drag'n'dropping on it. If that's what you're doing, then it's probably expected. The way the dropzone is supposed to be used is to either drag'n'drop on some element on your screen where you bound the dropzone or click to select a file and hit open.
